src/components/LaneBTCExplain.css

```
@@ -0,0 +1,62 @@
+.lanebtc-explain {
+  padding: 80px 0;
+  border-top: 1px solid rgba(158, 163, 175, 0.15);
```

Stylelint (17.2.0)

[error] Expected "0.15" to be "15%" (alpha-value-notation)

(alpha-value-notation)